f8a_worker/workers/libraries_io.py

```diff
@@ -45,7 +29,8 @@ def execute(self, arguments):
         if ecosystem == 'go':
             name = quote(name, safe='')
 
-        project = get_response(self.project_url(ecosystem, name))
+        project_url = self.configuration.libraries_io_project_url(ecosystem, name)
```

Comment:
Why did you move `libraries_io_project_url` to configuration module? It makes more sense to place it inside `utils` for me.

Reply:
Because it's basically a property of `F8AConfiguration`. I'd mark it `@property` if it had not arguments (ecosystem, name).

Reply:
Ok, thanks for an answer.